### PR TITLE
INS-1348: jet trees in memory

### PR DESCRIPTION
**Pull request by ruz**
no point for lights materials to store anything on disk, use in memory
trees.

### CODE REVIEW

File: ledger/storage/db.go [hunk 1]
```diff
@@ -114,6 +114,7 @@ type DB struct {
 	idlocker             *IDLocker
 	jetHeavyClientLocker *IDLocker
 
+	jetTree          map[core.PulseNumber]*jet.Tree
```
Comment (egorikas) [inline ledger/storage/db.go]: Why do we need it here?

Comment (egorikas) [inline ledger/storage/db.go]: Why do we need it here?

Comment (andreyromancev) [reply ledger/storage/db.go]: It should go to `jetStorage` struct

Comment (ruz) [reply ledger/storage/db.go]: done

**Pull request merged**